server/db.go

```diff
@@ -318,6 +318,12 @@ func (c *Server) fetchIssuers(issuerType string) (*[]Issuer, error) {
 }
 
 func (c *Server) FetchAllIssuers() (*[]Issuer, error) {
+	if c.caches != nil {
```

Comment:
is the issuer lookup so slow we need caching, or is this an optimization?

Reply:
This is an optimization because the issuers change rarely and this function gets called a lot.